app/main.py

```diff
@@ -1,3 +1,66 @@
-class Distance:
-    # Write your code here
-    pass
+class Distance:
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __add__(self, other: callable) -> callable:
+
+        if not isinstance(other, Distance):
+            return Distance(self.km + other.real)
+
+        return Distance(self.km + other.km)
+
+    def __iadd__(self, other: callable) -> callable:
+
+        if not isinstance(other, Distance):
+            self.km += other.real

```

Comment:
Why do you use real?

Reply:
Because I think that we use value as a complex number and use first 'real' part of this number. Now I changed that and use only "other", thanks